### PR TITLE
DOC CMS 4 Supported modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/developer-docs/issues/278

### CODE REVIEW

File: en/06_Project_Governance/06_Supported_Modules.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to [semantic versioning](https://semver.org/). They are covered by:
+- our [security release process](managing_security_issues) and 
+- our [major release policy](major_release_policy).
+
+These modules will be supported for the lifetime of Silverstripe CMS 4 in the provided major versions.
+
+[notice]
+This list is for Silverstripe CMS 4. For Silverstripe CMS 5 supported modules, see the [CMS 5 version of this page](/en/5/project_governance/supported_modules/).
+[/notice]
```
Comment (ChloeSartorelli) [inline en/06_Project_Governance/06_Supported_Modules.md]: There's already a banner mechanism for this.
```suggestion
```

File: en/06_Project_Governance/06_Supported_Modules.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
+title: Supported modules
+description: Modules which are commercially supported by Silverstripe
+---
+
+# Commercially supported modules
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to [semantic versioning](https://semver.org/). They are covered by:
+- our [security release process](managing_security_issues) and
```
Comment (ChloeSartorelli) [inline en/06_Project_Governance/06_Supported_Modules.md]:
```suggestion
- our [security release process](/contributing/managing_security_issues/) and 
```

File: en/06_Project_Governance/06_Supported_Modules.md
Comment (ChloeSartorelli) [inline en/06_Project_Governance/06_Supported_Modules.md:1]: Missing `silverstripe/security-extensions`, `silverstripe/activedirectory`, `cwp-themes/default` according to https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/06_Project_Governance/06_Supported_Modules.md (hunk 1 above)
Comment (emteknetnz) [reply en/06_Project_Governance/06_Supported_Modules.md]: Removed

File: en/06_Project_Governance/06_Supported_Modules.md
Comment (emteknetnz) [reply en/06_Project_Governance/06_Supported_Modules.md:1]: Have added security-extensions

https://packagist.org/packages/silverstripe/activedirectory and https://packagist.org/packages/cwp-themes/default are abandoned

File: en/06_Project_Governance/06_Supported_Modules.md (hunk 2 above)
Comment (emteknetnz) [reply en/06_Project_Governance/06_Supported_Modules.md]: Updated

**Comment by GuySartorelli** [comment]
@emteknetnz looks like Github didn't recieve the changes you pushed - can you please try force pushing them again?

**Comment by emteknetnz** [comment]
Changes now show. Had pushed changes to ccs repo, though for whatever reason this pull-request hadn't detected them. Have changed the base branch back and forth and that resolved issue.

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**